packages/application-shell/package.json

```diff
@@ -51,9 +51,9 @@
     "@commercetools-frontend/sentry": "13.6.0",
     "@commercetools-frontend/storage": "13.3.5",
     "@commercetools-frontend/url-utils": "13.3.5",
-    "@flopflip/launchdarkly-adapter": "2.5.0",
+    "@flopflip/launchdarkly-adapter": "2.6.0",
```

Comment:
Internally initialization changed to use another API from LD. While we can opt into getting an error thrown at us (we don't want yet) if initialzation fails.